cmd/reconciler/ip.go

```diff
@@ -29,13 +49,21 @@ func main() {
 	}
 	if err != nil {
 		_ = logging.Errorf("failed to create the reconcile looper: %v", err)
-		os.Exit(couldNotStartOrphanedIPMonitor)
+		if knownErrorCase(err) {
+			os.Exit(0)
+		} else {
+			os.Exit(couldNotStartOrphanedIPMonitor)
+		}
```

Comment:
would it make sense to add another helper func here ? This code is clearly repeated 3 times. I propose something like:

```golang
func tolerateKnownErrorCasesOrFail(err error, errorCodeToReturn int) {
    const pretendEverythingIsFine = 0

    if knownErrorCase(err) {
        os.Exit(pretendEverythingIsFine)
    } else {
        os.Exit(errorCodeToReturn)
    }
}
```

Reply:
Implemented! Good eye, agreed.